pages/common/figlet.md

```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Display large characters made up of ordinary screen characters.
```

Comment:
I think the description of the previous page was pretty much to the point. IMO "ASCII banners" is a better and clearer description than saying "large characters".

 Lets keep the original description.